Guardfile

```diff
@@ -1,11 +1,11 @@
 gemset = ENV['RVM_GEMSET'] || 'webmachine'
 gemset = "@#{gemset}" unless gemset.to_s == ''
 
-rvms = %W[ 1.8.7 1.9.2 1.9.3 jruby rbx ].map {|v| "#{v}#{gemset}" }
+rvms = %W[1.8.7 1.9.2 1.9.3 jruby rbx].map { |v| "#{v}#{gemset}" }
```

Comment:
If we aren't going to support these old versions of Ruby in CI, we should remove them from this file. Does anyone use `guard` anymore when working with the project?

Reply:
Usually, Guard is not part of my workflow.

Reply:
No

Reply:
I'm not sure if we should just remove the `Guardfile` or not. It looks coupled to RVM, and the list of rubies being used are all end of life (except jruby). It seems easier to remove it than work with it - I don't have RVM to test this properly.

Reply:
Alright - done in [this commit](https://github.com/webmachine/webmachine-ruby/commit/48125b18b1930935801a0df81c9eb075e5f076ec).